pymeasure/instruments/newport/esp300.py

```diff
@@ -220,10 +219,10 @@ def zero(self):
         self.write("DH")
 
     def wait_for_stop(self, delay=0, interval=0.05):
-        """ Blocks the program until the motion is completed. A further 
+        """ Blocks the program until the motion is completed. A further
         delay can be specified in seconds.
         """
-        self.write("WS%g" % delay*1e3)
+        self.write("WS%d" % (delay*1e3))
```

Comment:
Can you confirm that this should be an integer?

Reply:
Yes, the documentation for the instrument specifies this should be an integer. I have also tried it with the instrument itself and it has worked.

Reply:
Cool, thanks!